netci.groovy

```diff
@@ -55,15 +57,23 @@ def project = GithubProject
                 case 'OSX':
                     newJob.with{
                         steps{
-                            shell("./cibuild.sh --scope Test --target ${runtime}")
+                            buildCmd="./cibuild.sh --scope Test --target ${runtime}"
+                            if (runtime == "Mono") {
+                                buildCmd += " --host Mono"
```

Comment:
Isn't this implied by `--target`? I thought that was how we did it for `CoreCLR`.